content/applications/sales/sales/amazon_connector/manage.rst

```diff
@@ -1,53 +1,64 @@
-============================
-Manage Amazon orders in Odoo
-============================
+=======================
+Amazon order management
+=======================
 
-Synchronization of orders
-=========================
+Order synchronization
+=====================
+
+Orders are automatically fetched from Amazon, and synchronized in Odoo, at regular intervals.
 
-Orders are automatically fetched from Amazon and synchronized in Odoo at regular intervals.
 The synchronization is based on the Amazon status: only orders whose status has changed since the
-last synchronization are fetched from Amazon. For **FBA** (Fulfilled by Amazon), only **Shipped**
-and **Canceled** orders are fetched. For **FBM** (Fulfilled by Merchant), the same is done for
-**Unshipped** and **Canceled** orders. For each synchronized order, a sales order and a customer are
-created in Odoo if they are not yet registered.
+last synchronization are fetched from Amazon.
+
+For *FBA* (Fulfilled by Amazon), only *Shipped* and *Canceled* orders are fetched.
 
-When an order is canceled in Amazon and was already synchronized in Odoo, the corresponding sales
-order is automatically canceled in Odoo.
+For *FBM* (Fulfilled by Merchant), the same is done for *Unshipped* and *Canceled* orders. For each
+synchronization order, a sales order and customer are created in Odoo, if they aren't already
+registered.
 
 .. note::
-   To force the synchronization of an order whose status has not changed since the last
-   synchronization, activate the :ref:`developer mode <developer-mode>`, navigate to your Amazon
-   account and modify the date under :menuselection:`Orders Follow-up --> Last Order Sync`. Pick a
-   date anterior to the last status change of the order that you wish to synchronize and save.
+   When an order is canceled in Amazon, and was already synchronized in Odoo, the corresponding
+   sales order is automatically canceled in Odoo.
+
+Force synchronization
```

Comment:
FYI in Odoo 17 this section can change thanks to this feature: https://www.odoo.com/web#id=2806551&cids=1&menu_id=4720&action=333&active_id=4106&model=project.task&view_type=form

But we can keep a word if the user want to force the fetch of all changes since a specific date

Reply:
Okay. Good to know about 17 -- I'll be sure to keep that in mind. Thanks for letting me know. But, since this is targeting 16, can we leave this section as-is? @dalaOdoo

Reply:
it was just an FYI for 17 so yes